buildstock_fetch/main.py

```diff
@@ -1,11 +1,24 @@
 import json
-import os
```

Comment:
General comments:
- Remove the test output and combined metadata from the both the test output and the repo history, for the reasons max said.
- When the user is asked to select a release version, have the latest version appear first (it defaults to 1 not 1.1 for res_2022_tmy3)
- When the user is selecting the file types, would it possible to visually group them into two groups: 1. simulation inputs (HPXML, schedule) and 2. end-use load profiles (load curves, metadata)?
- Once you start downloading, we need to tell the user which file is being downloaded. Bonus points if you can render a progress bar using the `rich` library